src/lib/Persistor.js

```diff
@@ -2,35 +2,50 @@
 
 var _ = require('underscore'),
     Q = require('q'),
-    Class = require('class.extend');
+    Class = require('class.extend'),
+    low = require('lowdb'),
+    FileSync = require('lowdb/adapters/FileSync'),
+    adapter, db;
 
 module.exports = Class.extend({
 
    init: function() {
-      // TODO: actually read/write to a file
-      this.inMemoryStorage = {};
+      adapter = new FileSync('./store/db.json');
```

Comment:
Would this be better as `this.adapter`? (Same for `db` as well) As implemented, each time a new `Persistor` is created, the adapter and DB created with the previous instance will be clobbered.

I'm not too familiar with `lowdb`. So in the case we must have the database and adapter shared across instances of `Persistor`, it might be good to move this variable initialization to the var block at the top of the file.

Reply:
You're absolutely right, in fact I believe the same should be true of `d` if I'm not mistaken.